requirements.txt

```diff
@@ -1,4 +1,3 @@
-numpy # need to install first otherwise dependency issues
```

Comment:
Based on the commit message, I believe this change was to address #2750, right? 

But, I'm not sure this change to `requirements.txt` actually helps us here... I believe #2750 happens because `pip install numpy` installs the newest version, but `requirements-freeze.txt` pins an older version for reproducibility, so the newer version gets uninstalled to make room for the pinned version. 

Here, with simple `requirements.txt`, there ends up being two `pip install numpy` (no version pinning), so I imagine there won't be a double-install. Hence, no need to remove this line.

Reply:
Ah that's a good point. I didn't test how this interacts with `requirements-freeze.txt` because I was too deep in pip specs.

Another fix would be to version-pin numpy in requirements.txt to prevent freeze from being able to disagree and overriding it.

---

But my main thinking here was that we should only write down numpy in one place.

---

I'm also working on fixing this upstream in https://github.com/scikit-image/scikit-image/issues/4919

Reply:
Okay I was swayed. I've replaced removing this with a comment with all the backstory in install_sct. What do you think?

Reply:
That seems reasonable to me. :slightly_smiling_face: